doc/docs/master/deploy-manage/manage/use-transactions-to-run-multiple-commands.md

```diff
@@ -0,0 +1,221 @@
+storing all your data in one repository, often you prefer to separate
+it in individual repositories for organizational and logistics reasons.
+
+### Switching from Staging to Master Simultaneously
+
+If you use the [deferred processing](../../how-tos/deferred_processing/)
+model, you want to commit your changes often but do not want your
+pipeline to be triggered as often as you commit. When you want to postpone
+pipeline execution, you can create a staging and master branch in
+the same repository. You commit your changes to the staging branch and
+when needed, switch the `HEAD` of you master branch to a commit in the
+staging branch. To do this simultaneously, you can use transactions.
+
+For example, you have two repositories `data` and `parameters`, both
+of which have a `master` and `staging` branch. You commit your
+changes to the staging branch while your pipeline is subscribed to the
+master branch. To switch to these branches simultaneously, you can
+use transactions like this:
+
+```bash
+$ pachctl start transaction
+Started new transaction: 0d6f0bc3-37a0-4936-96e3-82034a2a2055
+$ pachctl pachctl create branch data@master --head staging
+Added to transaction: 0d6f0bc3-37a0-4936-96e3-82034a2a2055
+$ pachctl create branch parameters@master --head staging
+Added to transaction: 0d6f0bc3-37a0-4936-96e3-82034a2a2055
+$ pachctl finish transaction
+Completed transaction with 2 requests: 0d6f0bc3-37a0-4936-96e3-82034a2a2055
+```
+
+When you finish the transaction, both repositories switch to
+to the master at the same time which creates new commits in
+the corresponding `master` branches and the pipeline triggers
+one job to process the commits together.
+
+## Start and Finish Transactions
+
+To start a transaction, run the following command:
+
+```bash
+$ pachctl start transaction
+Started new transaction: 7a81eab5-e6c6-430a-a5c0-1deb06852ca5
+```
+
+This command generates a transaction object in the cluster and saves
+its ID in the local Pachyderm configuration file. By default, this file
+is stored at `~/.pachyderm/config.json`.
+
+!!! example
+    ```json hl_lines="9"
+    {
+       "user_id": "b4fe4317-be21-4836-824f-6661c68b8fba",
+       "v2": {
+         "active_context": "local-2",
+         "contexts": {
+           "default": {},
+           "local-2": {
+             "source": 3,
+             "active_transaction": "7a81eab5-e6c6-430a-a5c0-1deb06852ca5",
+             "cluster_name": "minikube",
+             "auth_info": "minikube",
+             "namespace": "default"
+           },
+    ```
+
+After you start a transaction, you can add supported commands, such
+as create a repo, create a branch, and so on, to th transaction. You
+can also start a commit within a transaction. If you close a transaction
+before you close a commit, the commands that you have specified inside
+the transaction and inside the
+commit will not be applied until you close all commits that you have
+started in that transaction.
```

Comment:
> If you close a transaction
before you close a commit, the commands that you have specified inside
the transaction and inside the
commit will not be applied until you close all commits that you have
started in that transaction.

I dont really understand what this is saying and I dont think it's accurate. There's no special logic around waiting on commits or something with transactions. Commits have the property that they dont do anything until you close them. 

Perhaps a better way to describe this is:

"All commands that are performed in a transaction are queued up and not executed against the actual cluster. One you finish the transaction, all queued command are executed atomically."